scilpy/reconst/raw_signal.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+
+import logging
+
+import numpy as np
+from dipy.core.sphere import Sphere
+from dipy.reconst.shm import sf_to_sh
+
+from scilpy.utils.bvec_bval_tools import (check_b0_threshold, identify_shells,
+                                          is_normalized_bvecs, normalize_bvecs,
+                                          DEFAULT_B0_THRESHOLD)
+
+
+def compute_sh_coefficients(dwi, gradient_table, sh_order=8,
+                            basis_type='descoteaux07', smooth=0.006,
+                            use_attenuation=False, force_b0_threshold=False,
+                            mask=None, sphere=None):
```

Comment:
Why is there a `sphere` argument if the calling script cannot change it? Is it because other scripts use this internally?

Reply:
Yes, our things in machine learning.

Reply:
Ok